src/store/getters/state-getters.ts

```diff
@@ -190,7 +191,8 @@ export const isEnableFilePay = (state: any, getters: any): boolean => {
   const step2Valid = state.stateModel.addPeopleAndRoleStep.valid
   const step3Valid = state.stateModel.createShareStructureStep.valid
   const step4Valid = state.stateModel.incorporationAgreementStep.valid
-  const step5Valid = state.stateModel.certifyState.valid && state.stateModel.incorporationDateTime.valid
+  const step5Valid = (state.stateModel.certifyState.valid && state.stateModel.certifyState.certifiedBy) &&
+    state.stateModel.incorporationDateTime.valid
```

Comment:
Hehe, "(A and B) and C" is the same as "(A and B and C)".

Reply:
good point! Will update

Reply:
updated in next commit